lib/phobos/listener.rb

```diff
@@ -127,6 +127,8 @@ def stop_listener
         @kafka_client.close
         log_info('Listener stopped', listener_metadata) if should_stop?
       end
+    rescue NoMethodError
+      log_error('Listener stopped failed', listener_metadata)
```

Comment:
maybe put the log as `Listener stop failed`, and also, it is possible to print out the NoMethodError message?